test/wasm_worker/thread_stack.c

```diff
@@ -31,13 +31,13 @@ void test_stack(int i)
 
 int main()
 {
-  EM_ASM(console.log(`Main thread stack base=0x${$0.toString(16)}, end=0x${$1.toString(16)}`), emscripten_stack_get_base(), emscripten_stack_get_end());
+  EM_ASM(out(`Main thread stack base=0x${$0.toString(16)}, end=0x${$1.toString(16)}`), emscripten_stack_get_base(), emscripten_stack_get_end());
```

Comment:
Do we want to standardize on using `emscripten_outf` rather than `EM_ASM(out(...))`?

Reply:
In general I think yes.  However, there are cases where `EM_ASM` is used instead to avoid pulling in the sprintf code form libc.. I'm not sure this is worth it for all most the most minimal tests.

I'll convert these ones here.